Sources/App/Commands/Common.swift

```diff
@@ -17,111 +17,125 @@ import PostgresKit
 import Vapor
 
 
-/// Update packages (in the `[Result<Joined<Package, Repository>, Error>]` array).
-///
-/// - Parameters:
-///   - client: `Client` object
-///   - database: `Database` object
-///   - results: `Joined<Package, Repository>` results to update
-///   - stage: Processing stage
-func updatePackages(client: Client,
```

Comment:
`updatePackages` (note the plural) was only called from analysis. Moving this into the `Analyze` namespace (which I'll update to `Analysis` at some point in the future) makes this explicit.